spec/lib/mcb/commands/apiv1/courses/find_spec.rb

```diff
@@ -10,7 +10,7 @@
     course2 = create(:course, subjects: [subject], site_statuses: [site_status])
 
 
-    url = 'http://localhost:3001/api/v1/2019/courses'
+    url = 'http://localhost:3001/api/v1/courses'
```

Comment:
Just to check, does this simulate the URL that UCAS is currently using? Or is it meant to be the primary way to access the API? To me this looks like it's simulating the URL that UCAS should have stored from the current version of the API.